checker/src/main/java/org/checkerframework/checker/signedness/SignednessAnnotatedTypeFactory.java

```diff
@@ -690,4 +693,43 @@ public boolean isRelevant(TypeMirror tm) {
     }
     return true;
   }
+
+  /**
+   * Requires that, when two formal parameter types are annotated with {@code @PolySigned}, the two
```

Comment:
The Javadoc of PolySigned should be updated, and probably its section of the manual.